documentation/UniversalRemotes.md

```diff
@@ -25,6 +25,28 @@ Make sure that every signal does what it's supposed to.
 
 If everything checks out, append these signals **to the end** of the [audio player universal remote file](/assets/resources/infrared/assets/audio.ir).
 
+## Projectors
```

Comment:
Most of this instruction is irrelevant, just check previous section

Reply:
Fixed